.github/workflows/nightly-dev-doc-build.yml

```diff
@@ -19,7 +19,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        image-tag: [v23.1.0, v23.2.0, v24.1.0]
+        image-tag: [v22.2.0, v23.1.0, v23.2.0, v24.1.0]
```

Comment:
I removed v22.2.0 in a previous PR. As we are packaging 22.2 files, it is better to have the doc tested - so reinstating it.